src/Tracker/safeWoopra.js

```diff
@@ -0,0 +1,26 @@
+function SafeWindow () {
+  Object.keys(window).forEach(key => {
+    Object.defineProperty(this, key,
+      { get: () => {
+        const value = window[key]
+        if (typeof value === "function"){
+          return value.bind(window)
+        }
+        if (key === "window") return this
+        return value
+      },
+        set: value=>{ window[key] = value }
+      }
+    )
+  })
+}
+SafeWindow.prototype = window;
+
+
+const windowKey = "onfidoSafeWindow8xmy484y87m239843m20";
```

Comment:
[Minor] As the hash is used multiple times, might be better to have it as a constant somewhere.

Reply:
I had tried it with a runtime constant and it hadn't worked. However, I hadn't though of making it a literal constant. It worked actually! Commit has been pushed. thanks.